src/main/java/org/opensearchmetrics/metrics/release/ReleaseInputs.java

```diff
@@ -9,7 +9,7 @@
 package org.opensearchmetrics.metrics.release;
 
 public enum ReleaseInputs {
-    VERSION_3_0_0("3.0.0", "open", "main", true),
+    VERSION_3_0_0("3.0.0-alpha1", "open", "main", true),
```

Comment:
Maybe this needs to be 3.0.0-beta1 now?

Reply:
Yes! @prudhvigodithi do we close this release and add a new for one beta?